src/contexts/Api/index.tsx

```diff
@@ -40,18 +45,18 @@ export const APIProvider = ({ children }: { children: React.ReactNode }) => {
   // api instance state
   const [api, setApi] = useState<ApiPromise | null>(null);
 
-  // network state
-  const defaultNetworkName =
```

Comment:
Turning it to [a module constant](https://github.com/Cardinal-Cryptography/aleph-zero-dashboard/pull/37/files#diff-de701a1090eab6a6ce998b10578ba8d16bd3d81a57a7caf3ea8d1d14141ab0a8R27), where it belongs.